compiler/rustc_target/src/spec/x86_64_pc_windows_gnu.rs

```diff
@@ -4,8 +4,8 @@ pub fn target() -> Target {
     let mut base = super::windows_gnu_base::opts();
     base.cpu = "x86-64".into();
     // Use high-entropy 64 bit address space for ASLR
+    base.add_pre_link_args(LinkerFlavor::Ld, &["-m", "i386pep", "--high-entropy-va"]);
```

Comment:
I was under the impression that `--high-entropy-va` was on by default for 64-bit executables in the win `ld` port. Seems good anyways.

Reply:
I'm not sure.
This was just a mechanical change to make flags in `LinkerFlavor::Ld` and `LinkerFlavor::Gcc` consistent.